com.reprezen.swagedit.core/META-INF/MANIFEST.MF

```diff
@@ -9,7 +9,7 @@ Require-Bundle: org.eclipse.ui,
  org.eclipse.ui.editors,
  org.eclipse.jface.text,
  org.dadacoalition.yedit,
- com.google.guava;bundle-version="15.0.0",
+ com.google.guava;bundle-version="21.0.0",
```

Comment:
@ghillairet , we use Guava 16 in the API Studio, see https://github.com/ModelSolv/RepreZen/commit/42f0f2eb0e954dd1c0516e945050ff54c579e901#r30206252 . How was version 21 of Guava chosen?

And... Do we still need Guava, even after migration to Java 8? :)